### ISSUE TITLE
Running Queries export to excel: invalid character exception

### ISSUE THREAD

**Issue by KorsG** [trimble-oss/dba-dash#799]
Hi

I am trying to Export a Running Queries snapshot to Excel and get the following exception (version 2.51.1):
![image](https://github.com/trimble-oss/dba-dash/assets/1583564/1163ca15-7d90-4dc5-afd1-2c638c3681a3)

Would it be possible to sanitize this character? :-)

Full exception details:
```
See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.Xml.XmlException: '', hexadecimal value 0x0B, is an invalid character. Line 11, position 4.
   at System.Xml.XmlTextReaderImpl.Throw(Exception e)
   at System.Xml.XmlTextReaderImpl.Throw(String res, String[] args)
   at System.Xml.XmlTextReaderImpl.ParseNumericCharRefInline(Int32 startPos, Boolean expand, StringBuilder internalSubsetBuilder, Int32& charCount, EntityType& entityType)
   at System.Xml.XmlTextReaderImpl.ParseCharRefInline(Int32 startPos, Int32& charCount, EntityType& entityType)
   at System.Xml.XmlTextReaderImpl.ParseText(Int32& startPos, Int32& endPos, Int32& outOrChars)
   at System.Xml.XmlTextReaderImpl.ParseText()
   at System.Xml.XmlTextReaderImpl.ParseElementContent()
   at DocumentFormat.OpenXml.OpenXmlLeafTextElement.Populate(XmlReader xmlReader, OpenXmlLoadMode loadMode)
   at DocumentFormat.OpenXml.OpenXmlCompositeElement.Populate(XmlReader xmlReader, OpenXmlLoadMode loadMode)
   at DocumentFormat.OpenXml.OpenXmlElement.ParseXml()
   at DocumentFormat.OpenXml.OpenXmlElement.MakeSureParsed()
   at DocumentFormat.OpenXml.OpenXmlCompositeElement.get_FirstChild()
   at DocumentFormat.OpenXml.OpenXmlCompositeElement.set_InnerXml(String value)
   at SpreadsheetLight.SLRstType.FromHash(String Hash)
   at SpreadsheetLight.SLDocument.AutoFitRowColumn(Boolean IsRow, Int32 StartIndex, Int32 EndIndex, Int32 MaxPixelLength)
   at SpreadsheetLight.SLDocument.AutoFitColumn(Int32 StartColumnIndex, Int32 EndColumnIndex, Double MaximumColumnWidth)
   at DBADashGUI.Common.SaveDataGridViewToXLSX(DataGridView& dgv, String path) in D:\a\dba-dash\dba-dash\DBADashGUI\Common.cs:line 249
   at DBADashGUI.Common.PromptSaveDataGridView(DataGridView dgv) in D:\a\dba-dash\dba-dash\DBADashGUI\Common.cs:line 155
   at DBADashGUI.Common.PromptSaveDataGridView(DataGridView& dgv) in D:\a\dba-dash\dba-dash\DBADashGUI\Common.cs:line 136
   at DBADashGUI.Performance.RunningQueries.TsExcel_Click(Object sender, EventArgs e) in D:\a\dba-dash\dba-dash\DBADashGUI\Performance\RunningQueries.cs:line 635
   at System.Windows.Forms.ToolStripButton.OnClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleMouseUp(MouseEventArgs e)
   at System.Windows.Forms.ToolStrip.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ScrollableControl.WndProc(Message& m)
   at System.Windows.Forms.ToolStrip.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, WM msg, IntPtr wparam, IntPtr lparam)


************** Loaded Assemblies **************
System.Private.CoreLib
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Private.CoreLib.dll
----------------------------------------
DBADash
    Assembly Version: 2.51.1.0
    Win32 Version: 2.51.1.0
    CodeBase: file:///E:/Data/Apps/DBADash/DBADash.dll
----------------------------------------
System.Runtime
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Runtime.dll
----------------------------------------
DBADashSharedGUI
    Assembly Version: 2.51.1.0
    Win32 Version: 2.51.1.0
    CodeBase: file:///E:/Data/Apps/DBADash/DBADashSharedGUI.dll
----------------------------------------
CommandLine
    Assembly Version: 2.9.1.0
    Win32 Version: 2.9.1.0
    CodeBase: file:///E:/Data/Apps/DBADash/CommandLine.dll
----------------------------------------
netstandard
    Assembly Version: 2.1.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/netstandard.dll
----------------------------------------
System.Windows.Forms
    Assembly Version: 6.0.2.0
    Win32 Version: 6.0.1322.60101
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.13/System.Windows.Forms.dll
----------------------------------------
System.Console
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Console.dll
----------------------------------------
System.Threading
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Threading.dll
----------------------------------------
System.Text.Encoding.Extensions
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Text.Encoding.Extensions.dll
----------------------------------------
System.Linq
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Linq.dll
----------------------------------------
System.Collections
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Collections.dll
----------------------------------------
System.ComponentModel.Primitives
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.ComponentModel.Primitives.dll
----------------------------------------
System.Windows.Forms.Primitives
    Assembly Version: 6.0.2.0
    Win32 Version: 6.0.1322.60101
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.13/System.Windows.Forms.Primitives.dll
----------------------------------------
System.Runtime.InteropServices
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Runtime.InteropServices.dll
----------------------------------------
System.Drawing.Primitives
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Drawing.Primitives.dll
----------------------------------------
System.Collections.Specialized
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Collections.Specialized.dll
----------------------------------------
System.Diagnostics.TraceSource
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Diagnostics.TraceSource.dll
----------------------------------------
PresentationFramework
    Assembly Version: 6.0.2.0
    Win32 Version: 6.0.1322.60105
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.13/PresentationFramework.dll
----------------------------------------
PresentationCore
    Assembly Version: 6.0.2.0
    Win32 Version: 6.0.1322.60105
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.13/PresentationCore.dll
----------------------------------------
WindowsBase
    Assembly Version: 6.0.2.0
    Win32 Version: 6.0.1322.60105
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.13/WindowsBase.dll
----------------------------------------
System.Xaml
    Assembly Version: 6.0.2.0
    Win32 Version: 6.0.1322.60105
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.13/System.Xaml.dll
----------------------------------------
DirectWriteForwarder
    Assembly Version: 6.0.2.0
    Win32 Version: 6,0,1322,60105 @Commit: 4ba96971b81698ec17061cbd6a59f752f5bfc3b5
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.13/DirectWriteForwarder.dll
----------------------------------------
System.Runtime.Extensions
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Runtime.Extensions.dll
----------------------------------------
System.Runtime.CompilerServices.VisualC
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Runtime.CompilerServices.VisualC.dll
----------------------------------------
System.Diagnostics.Debug
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Diagnostics.Debug.dll
----------------------------------------
Microsoft.Win32.Primitives
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/Microsoft.Win32.Primitives.dll
----------------------------------------
System.Collections.NonGeneric
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Collections.NonGeneric.dll
----------------------------------------
System.Drawing.Common
    Assembly Version: 7.0.0.0
    Win32 Version: 7.0.22.51805
    CodeBase: file:///E:/Data/Apps/DBADash/runtimes/win/lib/net6.0/System.Drawing.Common.dll
----------------------------------------
System.ComponentModel.EventBasedAsync
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.ComponentModel.EventBasedAsync.dll
----------------------------------------
System.Threading.Thread
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Threading.Thread.dll
----------------------------------------
Accessibility
    Assembly Version: 4.0.0.0
    Win32 Version: 6.0.1322.60101
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.13/Accessibility.dll
----------------------------------------
WindowsFormsIntegration
    Assembly Version: 6.0.2.0
    Win32 Version: 6.0.1322.60105
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.13/WindowsFormsIntegration.dll
----------------------------------------
DiffPlex.Wpf
    Assembly Version: 1.4.0.0
    Win32 Version: 1.4.0.0
    CodeBase: file:///E:/Data/Apps/DBADash/DiffPlex.Wpf.dll
----------------------------------------
Microsoft.Win32.Registry
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/Microsoft.Win32.Registry.dll
----------------------------------------
System.Private.Uri
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Private.Uri.dll
----------------------------------------
System.Configuration.ConfigurationManager
    Assembly Version: 7.0.0.0
    Win32 Version: 7.0.22.51805
    CodeBase: file:///E:/Data/Apps/DBADash/System.Configuration.ConfigurationManager.dll
----------------------------------------
System.Diagnostics.Process
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Diagnostics.Process.dll
----------------------------------------
System.Runtime.InteropServices.RuntimeInformation
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Runtime.InteropServices.RuntimeInformation.dll
----------------------------------------
System.Xml.ReaderWriter
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Xml.ReaderWriter.dll
----------------------------------------
System.Private.Xml
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Private.Xml.dll
----------------------------------------
System.Net.WebClient
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Net.WebClient.dll
----------------------------------------
System.Net.Primitives
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Net.Primitives.dll
----------------------------------------
System.Memory
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Memory.dll
----------------------------------------
System.ComponentModel.TypeConverter
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.ComponentModel.TypeConverter.dll
----------------------------------------
System.ComponentModel
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.ComponentModel.dll
----------------------------------------
UIAutomationTypes
    Assembly Version: 6.0.2.0
    Win32 Version: 6.0.1322.60105
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.13/UIAutomationTypes.dll
----------------------------------------
System.IO.Packaging
    Assembly Version: 7.0.0.0
    Win32 Version: 7.0.22.51805
    CodeBase: file:///E:/Data/Apps/DBADash/System.IO.Packaging.dll
----------------------------------------
System.ObjectModel
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.ObjectModel.dll
----------------------------------------
System.Runtime.CompilerServices.Unsafe
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Runtime.CompilerServices.Unsafe.dll
----------------------------------------
System.Numerics.Vectors
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Numerics.Vectors.dll
----------------------------------------
System.Threading.ThreadPool
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Threading.ThreadPool.dll
----------------------------------------
DiffPlex.Wpf.resources
    Assembly Version: 1.4.0.0
    Win32 Version: 1.4.0.0
    CodeBase: file:///E:/Data/Apps/DBADash/en/DiffPlex.Wpf.resources.dll
----------------------------------------
System.Windows.Extensions
    Assembly Version: 7.0.0.0
    Win32 Version: 7.0.22.51805
    CodeBase: file:///E:/Data/Apps/DBADash/runtimes/win/lib/net6.0/System.Windows.Extensions.dll
----------------------------------------
System.Collections.Concurrent
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Collections.Concurrent.dll
----------------------------------------
PresentationFramework.Aero2
    Assembly Version: 6.0.2.0
    Win32 Version: 6.0.1322.60105
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.13/PresentationFramework.Aero2.dll
----------------------------------------
UIAutomationProvider
    Assembly Version: 6.0.2.0
    Win32 Version: 6.0.1322.60105
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.13/UIAutomationProvider.dll
----------------------------------------
Microsoft.Win32.SystemEvents
    Assembly Version: 7.0.0.0
    Win32 Version: 7.0.22.51805
    CodeBase: file:///E:/Data/Apps/DBADash/runtimes/win/lib/net6.0/Microsoft.Win32.SystemEvents.dll
----------------------------------------
ICSharpCode.AvalonEdit
    Assembly Version: 6.3.0.90
    Win32 Version: 6.3.0.90
    CodeBase: file:///E:/Data/Apps/DBADash/ICSharpCode.AvalonEdit.dll
----------------------------------------
System.Text.RegularExpressions
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Text.RegularExpressions.dll
----------------------------------------
System.Net.Requests
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Net.Requests.dll
----------------------------------------
System.Net.WebHeaderCollection
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Net.WebHeaderCollection.dll
----------------------------------------
System.Resources.Extensions
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.13/System.Resources.Extensions.dll
----------------------------------------
System.Drawing
    Assembly Version: 6.0.2.0
    Win32 Version: 6.0.1322.60101
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.13/System.Drawing.dll
----------------------------------------
System.Runtime.Loader
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Runtime.Loader.dll
----------------------------------------
System.Runtime.Serialization.Formatters
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Runtime.Serialization.Formatters.dll
----------------------------------------
mscorlib
    Assembly Version: 4.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/mscorlib.dll
----------------------------------------
LiveCharts.WinForms
    Assembly Version: 0.9.7.1
    Win32 Version: 0.9.7.1
    CodeBase: file:///E:/Data/Apps/DBADash/LiveCharts.WinForms.dll
----------------------------------------
LiveCharts.Wpf
    Assembly Version: 0.9.7.0
    Win32 Version: 0.9.7
    CodeBase: file:///E:/Data/Apps/DBADash/LiveCharts.Wpf.dll
----------------------------------------
LiveCharts
    Assembly Version: 0.9.7.0
    Win32 Version: 0.9.7
    CodeBase: file:///E:/Data/Apps/DBADash/LiveCharts.dll
----------------------------------------
System
    Assembly Version: 4.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.dll
----------------------------------------
DiffPlex
    Assembly Version: 1.7.1.0
    Win32 Version: 1.7.1.0
    CodeBase: file:///E:/Data/Apps/DBADash/DiffPlex.dll
----------------------------------------
Microsoft.Web.WebView2.WinForms
    Assembly Version: 1.0.2151.40
    Win32 Version: 1.0.2151.40
    CodeBase: file:///E:/Data/Apps/DBADash/Microsoft.Web.WebView2.WinForms.dll
----------------------------------------
Microsoft.Web.WebView2.Core
    Assembly Version: 1.0.2151.40
    Win32 Version: 1.0.2151.40
    CodeBase: file:///E:/Data/Apps/DBADash/Microsoft.Web.WebView2.Core.dll
----------------------------------------
DBADashTools
    Assembly Version: 2.51.1.0
    Win32 Version: 2.51.1.0
    CodeBase: file:///E:/Data/Apps/DBADash/DBADashTools.dll
----------------------------------------
System.Security.Cryptography.Algorithms
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Security.Cryptography.Algorithms.dll
----------------------------------------
System.Security.Cryptography.Primitives
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Security.Cryptography.Primitives.dll
----------------------------------------
System.Xml.XmlSerializer
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Xml.XmlSerializer.dll
----------------------------------------
Microsoft.SqlServer.ConnectionInfo
    Assembly Version: 17.100.0.0
    Win32 Version: 17.100.18.0
    CodeBase: file:///E:/Data/Apps/DBADash/Microsoft.SqlServer.ConnectionInfo.dll
----------------------------------------
Newtonsoft.Json
    Assembly Version: 13.0.0.0
    Win32 Version: 13.0.3.27908
    CodeBase: file:///E:/Data/Apps/DBADash/Newtonsoft.Json.dll
----------------------------------------
System.Linq.Expressions
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Linq.Expressions.dll
----------------------------------------
System.Runtime.Numerics
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Runtime.Numerics.dll
----------------------------------------
System.Runtime.Serialization.Primitives
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Runtime.Serialization.Primitives.dll
----------------------------------------
System.Data.Common
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Data.Common.dll
----------------------------------------
System.Reflection.Emit.ILGeneration
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Reflection.Emit.ILGeneration.dll
----------------------------------------
System.Reflection.Emit.Lightweight
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Reflection.Emit.Lightweight.dll
----------------------------------------
System.Reflection.Primitives
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Reflection.Primitives.dll
----------------------------------------
Anonymously Hosted DynamicMethods Assembly
    Assembly Version: 0.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Private.CoreLib.dll
----------------------------------------
Microsoft.Data.SqlClient
    Assembly Version: 5.0.0.0
    Win32 Version: 5.12.23290.5
    CodeBase: file:///E:/Data/Apps/DBADash/runtimes/win/lib/net6.0/Microsoft.Data.SqlClient.dll
----------------------------------------
System.Security.Cryptography.ProtectedData
    Assembly Version: 7.0.0.0
    Win32 Version: 7.0.323.6910
    CodeBase: file:///E:/Data/Apps/DBADash/System.Security.Cryptography.ProtectedData.dll
----------------------------------------
System.Transactions.Local
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Transactions.Local.dll
----------------------------------------
System.Diagnostics.DiagnosticSource
    Assembly Version: 7.0.0.0
    Win32 Version: 7.0.423.11508
    CodeBase: file:///E:/Data/Apps/DBADash/System.Diagnostics.DiagnosticSource.dll
----------------------------------------
System.Diagnostics.Tracing
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Diagnostics.Tracing.dll
----------------------------------------
System.Diagnostics.StackTrace
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Diagnostics.StackTrace.dll
----------------------------------------
System.Security.Principal.Windows
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Security.Principal.Windows.dll
----------------------------------------
System.Security.Claims
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Security.Claims.dll
----------------------------------------
Microsoft.Identity.Client
    Assembly Version: 4.58.0.0
    Win32 Version: 4.58.0.0
    CodeBase: file:///E:/Data/Apps/DBADash/Microsoft.Identity.Client.dll
----------------------------------------
System.Text.Encoding.CodePages
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Text.Encoding.CodePages.dll
----------------------------------------
System.Runtime.Caching
    Assembly Version: 4.0.0.0
    Win32 Version: 7.0.22.51805
    CodeBase: file:///E:/Data/Apps/DBADash/runtimes/win/lib/net6.0/System.Runtime.Caching.dll
----------------------------------------
Microsoft.SqlServer.Server
    Assembly Version: 1.0.0.0
    Win32 Version: 1.0.0.0
    CodeBase: file:///E:/Data/Apps/DBADash/Microsoft.SqlServer.Server.dll
----------------------------------------
System.Reflection.Emit
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Reflection.Emit.dll
----------------------------------------
Microsoft.GeneratedCode
    Assembly Version: 1.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Private.Xml.dll
----------------------------------------
PresentationFramework-SystemData
    Assembly Version: 6.0.2.0
    Win32 Version: 6.0.1322.60105
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.WindowsDesktop.App/6.0.13/PresentationFramework-SystemData.dll
----------------------------------------
Humanizer
    Assembly Version: 2.14.0.0
    Win32 Version: 2.14.1.48190
    CodeBase: file:///E:/Data/Apps/DBADash/Humanizer.dll
----------------------------------------
SpreadsheetLight
    Assembly Version: 3.5.0.0
    Win32 Version: 3.5.0.0
    CodeBase: file:///E:/Data/Apps/DBADash/SpreadsheetLight.dll
----------------------------------------
DocumentFormat.OpenXml
    Assembly Version: 2.20.0.0
    Win32 Version: 2.20.0
    CodeBase: file:///E:/Data/Apps/DBADash/DocumentFormat.OpenXml.dll
----------------------------------------
System.IO.Compression
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.IO.Compression.dll
----------------------------------------
System.Reflection.Metadata
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Reflection.Metadata.dll
----------------------------------------
System.Collections.Immutable
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Collections.Immutable.dll
----------------------------------------
System.Diagnostics.FileVersionInfo
    Assembly Version: 6.0.0.0
    Win32 Version: 6.0.1322.58009
    CodeBase: file:///C:/Program%20Files/dotnet/shared/Microsoft.NETCore.App/6.0.13/System.Diagnostics.FileVersionInfo.dll
----------------------------------------

************** JIT Debugging **************
```

**Comment by DavidWiseman** [comment]
I have a fix for this - it will be included in the next release.

**Comment by DavidWiseman** [comment]
This is included in version 3.0.  Note, that there is an upgrade to .NET 8.  🚀
